Libraries/Opc.Ua.Gds.Server.Common/CertificateGroup.cs

```diff
@@ -175,25 +175,25 @@ public virtual async Task Init()
         public async virtual Task<X509CRL> RevokeCertificateAsync(
             X509Certificate2 certificate)
         {
-            Task<X509CRL> crl = RevokeCertificateAsync(
+            X509CRL crl = await RevokeCertificateAsync(
```

Comment:
@romanett I was checking on some warnings and something still lighted up with continuewith, so lets do it the safe way... want to make sure it works as expected.

Reply:
works as expected, I also see no case were this would lead to problems as when the first awaited task fails the exception will throw and the method will exit.